app/src/common/shared/org/mozilla/vrbrowser/telemetry/TelemetryWrapper.java

```diff
@@ -129,5 +174,24 @@ private static void browseEvent() {
         // TODO: Working on autocomplete result.
         event.queue();
     }
+
+    @UiThread
+    public static void addLoadToHistogram(String uri, Long newLoadTime) {
+        domainMap.add(UrlUtils.stripCommonSubdomains(URI.create(uri).getHost()));
```

Comment:
This seems to be striping just the common leading parts of domains but leaving all the user data and page info? If it is a domain map shouldn't it end at the first `/`?

Reply:
I am not sure I understand what you mean. For the example of `https://m.facebook.com/?refsrc=https%3A%2F%2Fwww.facebook.com%2F&_rdr`, `getHost()` will return `m.facebook.com` then `UrlUtils.stripCommonSubdomains ` helps us strip it to `facebook.com`.